gui/default/assets/lang/lang-en.json

```
@@ -258,12 +268,14 @@
     "More than a year ago": "More than a year ago",
     "Move to top of queue": "Move to top of queue",
     "Multi level wildcard (matches multiple directory levels)": "Multi level wildcard (matches multiple directory levels)",
+    "Name": "Name",
     "Never": "Never",
     "New Device": "New Device",
     "New Folder": "New Folder",
     "Newest First": "Newest First",
     "No": "No",
     "No File Versioning": "No File Versioning",
+    "No WebAuthn credentials configured.": "No WebAuthn credentials configured.",
```

Mention passkeys here as well? E.g.

```suggestion
"No passkeys (WebAuthn) configured.": "No passkeys (WebAuthn) configured.",
```

---

In that case I'll delete the `(WebAuthn)` part too. Done in dd0021b289e931cb52b7e746fe69f9c8790728d8.